.env.sample

```diff
@@ -34,3 +34,4 @@ REFERRAL_REJECTED_EMAIL_TEMPLATE_ID=xx
 REFERRAL_REJECTED_WITH_COMMENTS_EMAIL_TEMPLATE_ID=xx
 SYNONYMS={"ostrich": ["testsynonym"]}
 SIGNPOST_RECEIPT_EMAIL_TEMPLATE_ID=xxx
+TOKEN_RESET_IAT=
```

Comment:
From you answer to previous comment I understood that token resetting is a feature that's going to stay around.
Perhaps it would be convenient to set this to some default dummy value for it (when empty) not to interfere with development going forwards?

Reply:
Setting it to 0 wouldn't change how it works now, so could do that 👍